metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFactory.java

```diff
@@ -144,15 +154,15 @@ public ConnectorPartitionService getPartitionService() {
 
     @Override
     public String getName() {
-        return name;
+        return catalogName;
     }
 
     @Override
     public void stop() {
         try {
             client.shutdown();
         } catch (TException e) {
-            log.warn(String.format("Failed shutting down the catalog: %s", name), e);
+            log.warn(String.format("Failed shutting down the catalog: %s", catalogName), e);
```

Comment:
You don't have to embed a String.format inside the slf4j logs if you're just doing string replacement. It's redundant. you could change this to be `"Failed shutting down the catalog: {}", catalogName, e` and it would be the same.